lib/jarvis/commands/merge.rb

```diff
@@ -0,0 +1,15 @@
+require "clamp"
+
+module Jarvis module Command class Merge < Clamp::Command
+
+  banner "Merge a pull request into one or more branches."
+
+  parameter "URL", "The URL to merge"
+  parameter "BRANCHES ...", "The branches to merge"
+
+  def execute
+    puts "Merging: #{url} into #{branches_list}"
```

Comment:
I love the idea of using clamp to extract the important part of the command.
So the only remaining task to the command will be to to actually validate the value of the command.

This also mean that a complex command could be defined in their own gem?